docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -0,0 +1,216 @@
+
+.. Click the type of panel you want to create.
+
+.. Click the data source you want to visualize.
+
+. From the editor, add the <<aggregation-reference,aggregations>> you want to visualize, then click *Update*. 
++
+NOTE: For the *Date Histogram* to use an *auto interval*, the date field must match the primary time field of the index pattern.
+
+. To change the order, drag the aggregations along the editor. 
++ 
+[role="screenshot"]
+image:images/bar-chart-tutorial-3.png[Option to change the order of aggregations]
+
+. To customize the series colors, click the series in the legend, then select the color you want to use.
++ 
+[role="screenshot"]
+image:images/aggregation-based-color-picker.png[Color picker]
+
+[float]
+[[try-it-aggregation-based-panel]]
+=== Try it: Create an aggregation-based bar chart
+
+You collected data from your web server, and you want to visualize and analyze the data on a dashboard. To create a dashboard panel of the data, create
+a bar chart that displays the top five log traffic sources for every three hours.
+
+[float]
+==== Add the data and create the dashboard
+
+Add the sample web logs data that you'll use to create the bar chart, then create the dashboard.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample web logs*, click *Add data*.
+
+. Open the main menu, then click *Dashboard*.
+
+. Click *Create dashboard*.
+
+[float]
+==== Open and set up the aggregation-based bar chart
+
+Open the bar chart visualization builder and change the time range. 
+
+. On the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click *Aggregation based > Vertical bar*.
+
+.. On the *Choose a source* window, click *kibana_sample_data_logs*.
+
+. Change the <<set-time-filter, date time filter>>> to *Last 7 days*.
+
+[float]
+[[tutorial-configure-the-bar-chart]]
+==== Create the bar chart
+
+To create the bar chart, add a <<bucket-aggregations,bucket aggregation>>, then add the terms sub-aggregation to display the top values.
+
+. Add a *Buckets* aggregation.
+
+.. Click *Add*, then select *X-axis*.
+
+.. From the *Aggregation* dropdown, select *Date Histogram*.
+
+.. Click *Update*.
++
+[role="screenshot"]
+image:images/bar-chart-tutorial-1.png[Bar chart with sample logs data]
+
+. To show the top five log traffic sources, add a sub-bucket aggregation.
+
+.. Click *Add*, then select *Split series*.
++
+TIP: Aggregation-based panels support a maximum of three *Split series*.
+
+.. From the *Sub aggregation* dropdown, select *Terms*.
+
+.. From the *Field* dropdown, select *geo.src*.
+
+.. Click *Update*.
++
+[role="screenshot"]
+image:images/bar-chart-tutorial-2.png[Bar chart with sample logs data]
+
+[float]
+==== Save the panel
+
+Save and add the visualization panel to the dashboard.
+
+. From the toolbar, click *Save*.
+
+. Enter the *Title* and optional *Description*. 
+
+. From the *Tags* drop down, select any applicable tags. 
```

Comment:
Or you can create a tag.

Reply:
I would prefer to leave out specific tag steps and just link to step-by-step tag instructions.